client/components/filters/style.scss

```diff
@@ -10,7 +10,7 @@
 	display: flex;
 	margin-bottom: $gap-large;
 
-	@include breakpoint( '<1100px' ) {
+	@include breakpoint( '<1280px' ) {
```

Comment:
Not for this PR, but would it make sense at some point using `<wide` instead of `<1280px` in the CSS as we do in the JS? It looks weird to me that in the JS we use names for the breakpoints but in the CSS we use pixel values.

Reply:
That's possible, we can update `breakpoint` to use the names from the viewport, but I'm not sure it's really necessary. Designers would likely tell us "break at 1280px", then we'd have to go look up which name that is, and vice-versa. We could revisit it in the future, maybe.